lib/passaporte_web/service_account.rb

```diff
@@ -97,6 +97,22 @@ def persisted?
       @persisted == true
     end
 
+    # Activates an existing ServiceAccount. Returns true if successfull or false if not. In case of failure,
+    # it will fill the <tt>errors</tt> attribute with the reason for the failure to save the object.
+    #
+    # API method: <tt>PUT /organizations/api/activate/</tt>
+    #
+    # API documentation: http://myfreecomm.github.io/passaporte-web/pweb/api/account_manager.html#put-organizations-api-activate
+    def activate(identity)
+      response = Http.put("/organizations/api/activate/", {slug: self.plan_slug, identity: identity, global_account: self.uuid})
```

Comment:
Line is too long. [128/120]<br>Redundant curly braces around a hash parameter.<br>Redundant `self` detected.<br>Space inside { missing.<br>Space inside } missing.